tests/terraform/graph/checks/resources/EMRClusterHasSecurityConfiguration/main.tf

```
@@ -0,0 +1,59 @@
+resource "aws_emr_cluster" "pass" {
+  # ... other configuration ...
+
+  # EMR version must be 5.23.0 or later
+  release_label = "emr-5.24.1"
+
+  security_configuration = "securityGroupId"
```

```suggestion
  security_configuration = "example"
```
not so important, but it is definitely not a security group 😄